dataView/DataView.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+
+import {Component} from 'react';
+import {hoistComponent, elemFactory} from 'hoist/core';
+import {grid} from 'hoist/grid';
+import {GridModel} from 'hoist/grid';
+import {baseCol} from 'hoist/columns/Core';
+
+@hoistComponent()
+class DataView extends Component {
+
+    render() {
+        const {store, selection, contextMenuFn, itemFactory} = this.model,
+            {rowCls, rowHeight} = this.props;
+
+        return grid({
+            model: new GridModel({
+                store,
+                selection,
+                contextMenuFn,
+                columns: [
+                    baseCol({
+                        field: 'id',
+                        flex: 1,
+                        cellRendererFramework: (
+                            class extends Component {
+                                render() {
```

Comment:
Its a pain, but actually think we need to implement a refresh() method here as well on this Ad-hoc component, or we get warnings.    It can just return false.  Search for cellRendererFramework.   

Idea: Maybe we can make this a column enhancement where we automatically create this class if the user just passes a renderFunction with the name renderElement?

Reply:
Good catch, will do. Also, good idea about renderFunction in the column definition, although this is a slightly different use-case (we are passing the entire record, not the specified column field)